deploy/charts/alluxio/templates/csi/storageClass.yaml

```diff
@@ -0,0 +1,21 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+{{- $name := include "alluxio.name" . }}
+
+{{- if and .Values.csi.enabled .Values.csi.createStorageClass }}
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: {{ $name }}-csi
+provisioner: alluxio
+volumeBindingMode: Immediate
+{{- end -}}
```

Comment:
not sure if this mark means no end-of-line at last. is this expected?

Reply:
I didn't realize this would be a problem, but seems like a standard thing to do: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline.
I'll fix them in my PRs and be careful in the future.